src/main/java/org/opentripplanner/netex/mapping/NetexMapper.java

```diff
@@ -460,18 +461,36 @@ private void mapTripPatterns(Map<String, FeedScopedId> serviceIds) {
     for (JourneyPattern_VersionStructure journeyPattern : currentNetexIndex
       .getJourneyPatternsById()
       .localValues()) {
-      TripPatternMapperResult result = tripPatternMapper.mapTripPattern(journeyPattern);
+      tripPatternMapper
+        .mapTripPattern(journeyPattern)
+        .ifPresent(result -> {
```

Comment:
Calling a none-trivial code block inside a function chain is not clean. Consider making a private method for the code block or using Java language features like `if/for`.